### PR TITLE
Add helper script to run all check (for example before pushing)

### PR DESCRIPTION
**Pull request by kulla**
Hopefully I will open less PRs with errors in the future :smile: :see_no_evil: -> I want to have this in the codebase to make the live for other developers easier.

### CODE REVIEW

**Comment by elbotho** [comment]
> Hopefully I will open less PRs with errors in the future 😄 🙈 

Just as a side note: PRs with errors might feel bad, but are not a real problem imo.
That's why we have the checks running here.
Some companies even encourage devs not to slow down their machines by running checks and builds locally :)

Still like the helpers 👍

File: scripts/run_all_checks.sh [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+	if [ -n "$(git diff sqlx-data.json)" ]; then
+		error "You need to run sqlx:prepare and commit the changes in sqlx-data.json!"
+	fi
+}
+
+function run_pact_tests {
+	# Run db layer server in background
+	cargo run --quiet &
+	DB_LAYER_SERVER_ID=$!
+	sleep 20
```
Comment (inyono) [inline scripts/run_all_checks.sh:60]: This should probably use something like https://github.com/serlo/serlo.org-database-layer/blob/main/.github/wait-for-server.sh instead.

**Review by inyono — CHANGES_REQUESTED** [review]

**Comment by kulla** [comment]
@inyono Review suggestions implemented. Ready to be merged

**Review by inyono — APPROVED** [review]
LGTM. I added `prettier-plugin-sh` so that the bash scripts get also formatted. The only thing I noticed is that this still requires `yarn start` beforehand. We could possibly add that to the script, too (since `yarn start` won't do much if the server has already been started).

**Pull request merged**